src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,28 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Link } from 'react-router-dom'
 import "./LandingPage.scss"
+import { useQuery } from '@apollo/client'
+import { GET_WINNER } from '../../queries'
 
 const LandingPage = () => {
+  const [winner, setWinner] = useState(null);
+
+  useEffect(() => {
+    refetch()
+      .then(response => {
+        console.log(response.data.getWinner)
+        setWinner(response.data.getWinner)
+      })
+  }, [])
+
+  const { loading, error, data, refetch } = useQuery(GET_WINNER)
+  
+  const displayWinner = winner &&
+    <p className="winner">{data.getWinner.name} is the winner!!!!!!!</p>
+
```

Comment:
Thank you for implementing this! We can meet with BE folks and find out if the mutation is updating the admin.